### DISCUSSION TITLE
limit

### DISCUSSION THREAD

**Discussion by ttc111** [QuickShop-Community/QuickShop-Hikari#1259] [Ideas]
I want to ask some questions about QuickShop-Hikari. Is increasing the number of chest shops a player can create only possible through permissions? And is it possible to introduce a command later to increase or decrease this quantity?

Using commands provides a more effective way to adjust a player's shop quantity. Placing it in a menu makes it more convenient for players to purchase, and they can choose the quantity of shops they want to buy. The quantity is cumulative.

**Comment by Ghost-chu** [comment]
We have no plans to implement this feature in the foreseeable future.
